src/libs/api.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import { API_BASE_URL } from 'constants/url';
+
+async function getProfileImg() {
+  const response = await axios.get(`${API_BASE_URL}/profile-images/`);
```

Comment:
윤혁님이 따로 빼놓으셨던데 이건 머지 후 수정하면 좋을 것 같네요!

```suggestion
const instance = axios.create({ baseURL: API_BASE_URL });
const response = await instace.get(`/profile-images/`);
```
이런식으로요!